timesketch/lib/analyzers/domain.py

```diff
@@ -72,8 +73,22 @@ def run(self):
             return 'No domains to analyze.'
 
         domain_count_array = numpy.array(list(domain_counter.values()))
-        domain_20th_percentile = int(numpy.percentile(domain_count_array, 20))
-        domain_85th_percentile = int(numpy.percentile(domain_count_array, 85))
+        try:
+            domain_20th_percentile = int(numpy.percentile(domain_count_array, 20))
+        except IndexError:
+            logging.warning('Unable to calculate the 20th percentile.')
+            domain_20th_percentile = 0
+
+        try:
+            domain_85th_percentile = int(numpy.percentile(domain_count_array, 85))
+        except IndexError:
+            logging.warning('Unable to calculate the 85th percentile.')
+            high = domain_counter.most_common(1)
```

Comment:
high of what? :) Can we rename this to be a bit more descriptive?

Reply:
Oh, the linter is complaining about too long lines as well.

Reply:
```
timesketch/lib/analyzers/domain.py:77:0: C0301: Line too long (82/80) (line-too-long)
timesketch/lib/analyzers/domain.py:83:0: C0301: Line too long (82/80) (line-too-long)
```

Reply:
done... fixed both